### ISSUE TITLE
Multi-touch not working on Samsung Android 11 devices

### ISSUE THREAD

**Issue by cgarst** [moonlight-stream/moonlight-android#944]
Moonlight is suffering from an issue with Samsung Android 11 variants that's appears with registering multi-touch with the on screen virtual controller. For example, while holding a direction and A then, the direction goes away when you pressing A. This is becoming a broadly known issue and a fix can be seen in the Lemuroid issue and commit below. I have verified the issue with moonlight on a Z Fold 2 on Android 11.0, though it supposedly affects most Samsung devices on 11.0.

https://github.com/Swordfish90/Lemuroid/issues/178#issuecomment-782883990

**Comment by J3DAJ** [comment]
Same issue on Samsung Galaxy S21 Ultra, Android 11, OneUI 3.1

**Comment by dnlmlr** [comment]
I can also confirm the same issue on Samsung Tab S7. 
- Android 11
- OneUI 3.1
- Moonlight 9.8.6

**Comment by dnlmlr** [comment]
It seems like this is not a problem with Samsungs Android 11 in general, but instead the problem is introduced by the Samsung `Game Booster` feature. That is also the conclusion that the user GbBrou came to in the [issue thread](https://github.com/Swordfish90/Lemuroid/issues/178#issuecomment-785145153) on Lemuriod that was linked here in the initial issue.

Specifically the issue is the `Graphics Quality` setting in the `Game Booster Plus` plugin for `Game Booster`. If it is not set to 100, we get the multitouch problems. That indicates that the lowered `Graphics Quality` might result in a somehow changed resultion mapping or something along those lines. I have never used or enabled the `Game Booster Plus` plugin, so that seems to affect the system even if it is not even enabled.

TLDR: The issue can currently be mitigated by enabling the `Game Booster Plus` plugin for `Game Booster` and changing the `Graphics Quality` setting to 100 for the Moonlight App.
